salemWitchTrials/.htaccess

```diff
@@ -0,0 +1 @@
+Redirect 301 /salemwitchtrials https://github.com/salem-witCHes/salemWitchTrials
```

Comment:
URLs are case sensitive and `Redirect` matches the request path exactly.  Need to change to this:
```diff
-Redirect 301 /salemwitchtrials https://github.com/salem-witCHes/salemWitchTrials
+Redirect 301 /salemWitchTrials https://github.com/salem-witCHes/salemWitchTrials
```
Alternatively, look at using `mod_rewrite` style with `RewriteRule`.